pyccel/decorators.py

```diff
@@ -26,8 +26,8 @@
 def lambdify(f):
 
     args = f.__code__.co_varnames
-    from sympy import symbols
-    args = symbols(args)
+    from sympy import Symbol
+    args = Symbol(args)
```

Comment:
Why did you change these lines? Are you sure that `sympy.Symbol` can return multiple objects like `sympy.symbols`?

Reply:
ah true its not the same i will correct it

Reply:
done in 5a525fe

Reply:
Thanks!